src/test/java/lifeline/command/CommandHandlerTest.java

```diff
@@ -0,0 +1,321 @@
+package lifeline.command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.File;
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import lifeline.exception.LifelineException;
+import lifeline.storage.Storage;
+import lifeline.task.Task;
+import lifeline.task.TaskList;
+import lifeline.task.ToDo;
+import lifeline.ui.Ui;
+
+public class CommandHandlerTest {
```

Comment:
I like the way you named the tests as it makes it easier to know which method is tested and the test scenario, which follows the coding standard well.

Reply:
Thank you!